var/spack/repos/builtin/packages/openvkl/package.py

```diff
@@ -24,18 +25,23 @@ class Openvkl(CMakePackage):
     version("1.0.0", sha256="81ccae679bfa2feefc4d4b1ce72bcd242ba34d2618fbb418a1c2a05d640d16b4")
     version("0.13.0", sha256="974608259e3a5d8e29d2dfe81c6b2b1830aadeb9bbdc87127f3a7c8631e9f1bd")
 
-    depends_on("embree@3.13.0:3")
+    depends_on("embree@4", when="@1.3.2:")
+    depends_on("embree@3.13.0:3", when="@:1.3.1")
     depends_on("embree@3.13.1:", when="@1.0.0:")
     depends_on("ispc@1.15.0:", type=("build"))
     depends_on("ispc@1.16.0:", when="@1.0.0:", type=("build"))
     depends_on("ispc@1.18:", when="@1.3:", type=("build"))
     depends_on("rkcommon@1.6.1:")
     depends_on("rkcommon@1.7.0:", when="@1.0.0:")
     depends_on("rkcommon@1.8.0:", when="@1.1:")
+    depends_on("rkcommon@:1.10.0", when="@:1.3.1")
+    depends_on("rkcommon@1.11.0:", when="@1.3.2:")
     depends_on("tbb")
 
     def cmake_args(self):
         args = [
+            # otherwise, openvkl 1.3.2 tries to install its headers into /openvkl
+            self.define("CMAKE_INSTALL_INCLUDEDIR", "{0}/include".format(self.spec.prefix)),
```

Comment:
FWIW. You could use an f-string to format the prefix.

Reply:
Thanks for the suggestion! Applied it and rebased everything for a hopefully clean CI run.